spec/requests/sitemap_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+    @work = create(:work, collection: @collection)
+    @page = create(:page, work: @work, status: 'transcribed')
+  end
+
+  describe "GET /sitemap.xml" do
+    it "returns valid XML sitemap index" do
+      get "/sitemap.xml"
+      
+      expect(response).to have_http_status(:success)
+      expect(response.content_type).to include("application/xml")
+      
+      # Parse XML to validate structure
+      doc = Nokogiri::XML(response.body)
+      expect(doc.at_xpath("//sitemapindex")).to be_present
+      expect(doc.xpath("//sitemap").count).to eq(3) # collections, works, pages
+    end
+  end
+
+  describe "GET /sitemap_collections.xml" do
+    it "returns collections sitemap with correct URLs" do
+      get "/sitemap_collections.xml"
+      
+      expect(response).to have_http_status(:success)
+      expect(response.content_type).to include("application/xml")
+      
+      doc = Nokogiri::XML(response.body)
+      urls = doc.xpath("//url/loc").map(&:text)
+      
+      expected_url = "#{request.protocol}#{request.host_with_port}/#{@owner.slug}/#{@collection.slug}"
+      expect(urls).to include(expected_url)
+    end
+  end
+
+  describe "GET /sitemap_works.xml" do
+    it "returns works sitemap with correct URLs" do
+      get "/sitemap_works.xml"
+      
+      expect(response).to have_http_status(:success)
+      expect(response.content_type).to include("application/xml")
+      
+      doc = Nokogiri::XML(response.body)
+      urls = doc.xpath("//url/loc").map(&:text)
+      
+      expected_url = "#{request.protocol}#{request.host_with_port}/#{@owner.slug}/#{@collection.slug}/#{@work.slug}"
+      expect(urls).to include(expected_url)
+    end
+  end
+
+  describe "GET /sitemap_pages.xml" do
+    it "returns pages sitemap with correct URLs" do
+      get "/sitemap_pages.xml"
+      
+      expect(response).to have_http_status(:success)
+      expect(response.content_type).to include("application/xml")
+      
+      doc = Nokogiri::XML(response.body)
+      urls = doc.xpath("//url/loc").map(&:text)
+      
+      expected_url = "#{request.protocol}#{request.host_with_port}/#{@owner.slug}/#{@collection.slug}/#{@work.slug}/display/#{@page.id}"
+      expect(urls).to include(expected_url)
+    end
+  end
+
+  describe "restricted collections" do
+    it "excludes restricted collections from sitemap" do
+      restricted_collection = create(:collection, owner: @owner, is_active: true, restricted: true)
+      
+      get "/sitemap_collections.xml"
+      
+      doc = Nokogiri::XML(response.body)
+      urls = doc.xpath("//url/loc").map(&:text)
+      
+      restricted_url = "#{request.protocol}#{request.host_with_port}/#{@owner.slug}/#{restricted_collection.slug}"
+      expect(urls).not_to include(restricted_url)
+    end
+  end
+
+  describe "blank pages" do
+    it "excludes blank pages from sitemap" do
+      blank_page = create(:page, work: @work, status: 'blank')
```

Comment:
In addition to blank pages--which are pages which a user has intentionally marked as blank--it should also exclude pages in the default state.